### PR TITLE
replace get_coupon_codes with get_coupon_codes due to deprecation

### PR DESCRIPTION
**Pull request by YasienDwieb**

### CODE REVIEW

File: includes/class-dropshipping-woocommerce-orders.php [hunk 1]
```diff
@@ -516,7 +516,7 @@ public function knawat_dropshipwc_create_sub_order_shipping( $parent_order, $ord
      * @return type
      */
     public function knawat_dropshipwc_create_sub_order_coupon( $parent_order, $order_id, $product_ids ) {
-        $used_coupons = $parent_order->get_used_coupons();
+        $used_coupons = $parent_order->get_coupon_codes();
```
Comment (iamdharmesh) [inline includes/class-dropshipping-woocommerce-orders.php:519]: Please keep supporting old versions as well. like 
```
if(WC version less than 3.7){
    $used_coupons = $parent_order->get_used_coupons();
}else{
    $used_coupons = $parent_order->get_coupon_codes();
}
```

**Review by iamdharmesh — CHANGES_REQUESTED** [review]

File: includes/class-dropshipping-woocommerce-orders.php (hunk 1 above)
Comment (YasienDwieb) [reply includes/class-dropshipping-woocommerce-orders.php:519]: Yes, you are absolutely correct. I will do that.
Thanks

**Review by iamdharmesh — APPROVED** [review]

**Review by hamusthekiller — APPROVED** [review]

File: includes/class-dropshipping-woocommerce-orders.php (hunk 1 above)
Comment (YasienDwieb) [reply includes/class-dropshipping-woocommerce-orders.php]: Done, please check again.

Comment (iamdharmesh) [reply includes/class-dropshipping-woocommerce-orders.php]: Thanks a lot YasienDwieb.

**Pull request merged**